api/v1beta1/cinderapi_types.go

```diff
@@ -137,3 +141,4 @@ func init() {
 func (instance CinderAPI) IsReady() bool {
 	return instance.Status.ReadyCount >= 1
 }
+
```

Comment:
Unnecessary new line at the end of the file

Reply:
Done